securedrop/journalist_templates/account_edit_hotp_secret.html

```diff
@@ -1,11 +1,10 @@
 {% extends "base.html" %}
 {% block body %}
+<h1>{{ gettext('Change Secret') }}</h1>
 <form method="post">
   <input name="csrf_token" type="hidden" value="{{ csrf_token() }}">
-  <p>
-    <label for="otp_secret">{{ gettext('Change Secret') }}</label>
-    <input name="otp_secret" type="text" placeholder="{{ gettext('HOTP Secret') }}"><br>
-  </p>
-  <button type="submit">{{ gettext('CONTINUE') }}</button>
+  <label for="otp_secret">{{ gettext('Change Secret') }}</label>
+  <input name="otp_secret" type="text" placeholder="{{ gettext('HOTP Secret') }}">
+  <button type="submit" aria-label="{{ gettext('Continue') }}">{{ gettext('CONTINUE') }}</button>
```

Comment:
Thanks.  I've expanded a number of `aria-label`s in 39954528f.  In other places, the change of capitalization remains to prevent screen-readers from mistaken all-caps words for initialisms.